### PR TITLE
Updates to reflect changes in the provisioning script

### PR DESCRIPTION
**Pull request by JohnPWSharp**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@JohnPWSharp__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: docs/building-and-deploying-the-sample-solution.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
...
+This document describes how to build and run the sample IoT solution. The solution simulates a large number of devices sending environmental readings for apartments in a smart building. The IoT system supports a number of configurations implementing a *warm path* that tracks events as they occur and performs some intermediate processing (this processing will take a finite time and the results might not be immediately available), and a *cold path* where the data is stored for further ad-hoc analysis. Click [here](https://github.com/mspnp/iot-journey/blob/master/docs/overview-of-architecture-and-components.md) for more information about these configurations.
```
Comment (bennage) [inline docs/building-and-deploying-the-sample-solution.md]: Do we want to remove the path language here as well?

File: docs/building-and-deploying-the-sample-solution.md [hunk 2]
```diff
@@ -102,7 +100,7 @@ The following sections describe these steps in more detail.
 
 ### Verifying the Azure Storage Configuration
 
-The provisioning script uses Azure blob storage to store data after it has been captured by the hot path for further analysis. The cold path also writes data to blob storage. Furthermore, the warm path retrieves reference data used for generating reporting data from blob storage. Perform the following steps to verify the configuration Azure storage:
+The provisioning script generates Azure blob storage to the raw event store data after it has been captured by Stream Analytics for further analysis. A separate Stream Analytics job retrieves reference data used for generating reporting data from blob storage. Perform the following steps to verify the configuration Azure storage:
```
Comment (bennage) [inline docs/building-and-deploying-the-sample-solution.md]: > The provisioning script generates Azure blob storage to the raw event store data after it has been captured by Stream Analytics for further analysis.

This is confusing. I'm not sure that I follow.

Comment (bennage) [inline docs/building-and-deploying-the-sample-solution.md]: > A separate Stream Analytics job retrieves reference data used for generating reporting data from blob storage. 

I think that "utilizies" or something similiar with be better than "retrieves" here.

File: docs/building-and-deploying-the-sample-solution.md (hunk 1 above)
Comment (JohnPWSharp) [inline docs/building-and-deploying-the-sample-solution.md]: Done

File: docs/building-and-deploying-the-sample-solution.md (hunk 2 above)
Comment (JohnPWSharp) [inline docs/building-and-deploying-the-sample-solution.md]: Both comments addressed.

**Pull request merged**